### ISSUE TITLE
Ошибка - Access denied 1020 (Решение)

### ISSUE THREAD

**Issue by Shark-vil** [GameServerManagers/LinuxGSM#3853]
https://github.com/GameServerManagers/LinuxGSM/issues/3850
--
Автор очень круто "[сражается с режимом](https://twitter.com/dangibbsuk/status/1498656546850496515?s=21&t=1L8O4vMRjG32Z1YuV9OiQQ)" другой страны, путём запрета российских IP-адресов (нет). 
Для тех кому лень включать VPN, я сделал копии сайтов с инструкциями и документацией. Не всё идеально работает, но можете пользоваться:
https://lgsm.vincy.ru/
https://lgsm.vincy.ru/docs/

Также форкнул репозиторий с последними обновлениями и удалил все связывающие ссылки от оригинала, можете использовать как безопасную альтернативу:
https://github.com/Shark-vil/LinuxGSM

Да, вы всё также можете использовать установщик из оригинального репозитория, просто пишите:
> wget -O linuxgsm.sh https://raw.githubusercontent.com/GameServerManagers/LinuxGSM/master/linuxgsm.sh && chmod +x linuxgsm.sh

Однако учитывая такие мувы от разработчика, советую хорошо проверять последние обновления. Вдруг он решит побороться с файлами на ваших серверах.

---
## Just a translation for English speakers

The author is very cool "[fighting the regime](https://twitter.com/dangibbsuk/status/1498656546850496515?s=21&t=1L8O4vMRjG32Z1YuV9OiQQ)" of another country, by banning Russian IP addresses (no). 
For those who are too lazy to turn on VPN, I made copies of sites with instructions and documentation. Not everything works perfectly, but you can use it:
https://lgsm.vincy.ru/
https://lgsm.vincy.ru/docs/

I also forked repository with latest updates and removed all linking links from the original, you can use it as a safe alternative:
https://github.com/Shark-vil/LinuxGSM

Yes, you can still use the installer from the original repository, just write:
> wget -O linuxgsm.sh https://raw.githubusercontent.com/GameServerManagers/LinuxGSM/master/linuxgsm.sh && chmod +x linuxgsm.sh

However, given such actions from the developer, I advise you to carefully check the latest updates. In case he decides to fight against files on your servers.

---
## Appeal to the author:
This whole situation is unpleasant to all adequate people. You apparently don't realize you're fighting the wrong people, lol.

**Comment by Shark-vil** [comment]
> Duplicate of #3851

No, the point is different)

**Comment by Claiyc** [comment]
> > Duplicate of #3851
> 
> No, the point is different)

Same topic, same origin of discussion. So please continue in the linked discussion, thank you.

**Comment by dgibbs64** [comment]
Like many open-source projects, I have chosen to show solidarity with Ukraine. I have done this by carrying out a basic "sanction" by blocking access to Russia and Belarus. Once Vladimir Putin stops invading Ukraine I will be happy to unblock. If you are unhappy about limited access to websites and have concerns about the tragic loss of life of Ukrainians and Russians and I recommend writing to your local Russian MP.

**Comment by Shark-vil** [comment]
> Like many open-source projects, I have chosen to show solidarity with Ukraine. I have done this by carrying out a basic "sanction" by blocking access to Russia and Belarus. Once Vladimir Putin stops invading Ukraine I will be happy to unblock. If you are unhappy about limited access to websites and have concerns about the tragic loss of life of Ukrainians and Russians I recommend writing to your local Russian MP.

I think it's easy to talk about overthrowing a regime with which you are superficially familiar. Think people decide something?)

Okay, this discussion could go on forever. All the best. I really hope that you will not stoop to introducing viruses into software and DDoS attacks.

**Comment by dgibbs64** [comment]
I did not suggest you overthrow the Russian government. I suggested writing to your local MP to express your concerns, which is something anyone in the UK can do without fear. I have done it myself a few times. 

I would never introduce viruses to my software or DDoS anyone that seems like an act of self-sabotage and chilsih. Plus I am far too busy for that nonsense. 

All the best.

**Comment by MeidoNoHitsuji** [comment]
> I did not suggest you overthrow the Russian government. I suggested writing to your local MP to express your concerns, which is something anyone in the UK can do without fear. I have done it myself a few times.
> 
> I would never introduce viruses to my software or DDoS anyone that seems like an act of self-sabotage and chilsih. Plus I am far too busy for that nonsense.
> 
> All the best.

Do not confuse the political system and the mentality of the citizens of Great Britain and Russia. If you watched Navalny's analyzes and the video of the "vdud" channel, then you can understand that nothing depends on ordinary citizens here. Our "letters to deputies" are, firstly, "a bunch in a puddle" and, secondly, a reason to hang an article on discrimination on us, for which we are imprisoned. Roughly speaking, there is no freedom of speech. (By the way, they can put me in jail for this :))
I understand that your blocking is at least some way to express your position, but with these antics of yours you will get 2 results. The first is the outflow of Russian users from your development (to be honest, I don’t care about it myself, I raise my custom game servers myself), and the second is the creation of such forks. So your blocking is as meaningless as possible.

(Translated by Google Translate, so I apologize for possible errors)

**Comment by animegrafmays** [comment]
> Like many open-source projects, I have chosen to show solidarity with Ukraine. I have done this by carrying out a basic "sanction" by blocking access to Russia and Belarus. Once Vladimir Putin stops invading Ukraine I will be happy to unblock. If you are unhappy about limited access to websites and have concerns about the tragic loss of life of Ukrainians and Russians I recommend writing to your local Russian MP.

glad I chose not to donate, I don't support open source projects shoehorning in personal politics and nobody else should either

**Comment by dgibbs64** [comment]
![R.gif](https://user-images.githubusercontent.com/4478206/179199181-9edbdbde-683f-40f7-98a9-ded5a1cd852f.gif)
